Extension/src/nativeStrings.json

```diff
@@ -31,8 +31,8 @@
     "formatting_failed": "Formatting failed:",
     "database_add_file_failed": "Unable to add file to database, error = {0}: {1}",
     "reset_timestamp_failed": "Failed to reset timestamp during abort, error = {0}: {1}",
+    "symbol_add_failed": "",
```

Comment:
Can't we just remove this line entirely?

Reply:
As long as the native file is regenerated, the indexes should line up.

Reply:
It causes changes to the indexes, so retranslation.

Reply:
It would not cause retranslation.  We export an xlf file that is used for translations, and it does not include these indexes, just the label names.

Reply:
Okay, I removed the empty strings.